vtex/pickup-points-modal
# Add behavior to when going back to the list from details to scroll to…

… the last pickup point selected

#### What is the purpose of this pull request?
Ditto.

#### What problem is this solving?
Closes [story](https://app.clubhouse.io/vtex/story/23529/posi%C3%A7%C3%A3o-na-lista-de-pickup-points-se-perde-ao-voltar-da-tela-de-detalhamento-do-ponto)

#### How should this be manually tested?
1. Add [items to cart](https://fernando--drimjuguetes.myvtex.com/checkout/cart/add?&workspace=fernando&sku=63525&qty=1&seller=1&sc=1)
2. Search for pickups in `Barcelona`
3. Select a pickup point in the list
4. Clicking the back link should return to the same point the previous pickup point was selected

#### Screenshots or example usage
n/a
#### Types of changes
